wsd/DocumentBroker.cpp

```diff
@@ -1266,13 +1269,19 @@ void DocumentBroker::handleSaveResponse(const std::string& sessionId, bool succe
     const std::string oldName = _storage->getRootFilePathToUpload();
     const std::string newName = _storage->getRootFilePathUploading();
 
+    // Rename even if no new save, in case we have an older version.
     if (rename(oldName.c_str(), newName.c_str()) < 0)
     {
         // It's not an error if there was no file to rename, when the document isn't modified.
         const auto onrre = errno;
-        LOG_DBG("Failed to renamed [" << oldName << "] to [" << newName << "] ("
-                                      << Util::symbolicErrno(onrre) << ": " << std::strerror(onrre)
-                                      << ')');
+        if (success || onrre != ENOENT)
+            LOG_ERR("Failed to rename [" << oldName << "] to [" << newName << "] ("
```

Comment:
I guess you wanted to print something different here than the else block will :)

Reply:
The log-level is different. I could make it more complex to reduce duplication, but the number of lines and readability will probably not improve.

I thought of adding more info, or to differentiate, but for some reason couldn't find anything to change. So I left them identical.